Dockerfile.ocp

```diff
@@ -3,9 +3,11 @@ FROM ubi8
 # We don't need the deps that rhosp-director-images-ipa pulls in
 # use rpm to install without them to keep the image size down as
 # much as possible
-RUN yum install -y rhosp-director-images-ipa-$(uname -m) --downloadonly --downloaddir=/tmp/packages && \
-    rpm -i --nodeps /tmp/packages/rhosp-release-* /tmp/packages/rhosp-director-images-ipa-* && \
+RUN dnf download rhosp-director-images-ipa-$(uname -m) --destdir /tmp/packages && \
+    rpm -i --nodeps /tmp/packages/rhosp-director-images-ipa-* && \
```

Comment:
I'm a bit concerned about using rpm --nodeps directly: if rhosp-director-images really doesn't require these packages why are they dependencies?

Reply:
@stbenjam the rhosp-director-images package contains the images that we need for the inspection process, the only thing that happens during its installation is extracting those images and putting them in the correct location, plus calling a script to create/update the links to the latest images.
The script is contained in the rhosp-release package, which is a hard dependency of rhosp-director-images, and the only dependency that it actually needs.
While using yum, even with the --downloadonly option, the presence of the dependency is mandatory, using dnf download pulls only the rhosp-director-images package.
With this change, we remove the need to have the rhosp-release package installed, we don't create the links, but we move the images to the correct location manually during the build process.

Reply:
That makes sense to me, we just want the contents of the rpm, nothing else...... (We could always just use rpm2cpio....)

Reply:
I thought about rpm2cpio, but using rpm to install the package is convenient because the location of the images is an absolute path